chain/jsonrpc/res/epoch_info.html

```diff
@@ -71,6 +71,48 @@
             return bytes.toFixed(dp) + ' ' + units[u];
         }
 
+        function process_validator_info(validator_info) {
+            let total_stake = 0;
+            validator_info.current_validators.forEach(validator => {
+                total_stake += parseInt(validator.stake);
+            })
+            $('.js-tbody-curepoch').append("Total stake " + total_stake)
+            validator_info.current_validators.forEach(validator => {
+                let row = $('<tr>');
+                console.log(validator.stake + "validator stake\n");
+                row.append($('<td>').append(validator.account_id));
+                row.append($('<td>').append((100 * validator.stake / total_stake).toFixed(2) + "%"));
+                row.append($('<td>').append(validator.shards.join(",")));
+                row.append($('<td>').append(validator.num_expected_blocks - validator.num_produced_blocks));
+                row.append($('<td>').append(validator.num_expected_blocks));
+                row.append($('<td>').append(validator.num_expected_chunks - validator.num_produced_chunks));
+                row.append($('<td>').append(validator.num_expected_chunks));
```

Comment:
let's add a column with the percent, that we're using in prod for kickout (num_producer / num_expected).